fe/fe-core/src/main/java/com/starrocks/sql/parser/ParsingException.java

```diff
@@ -0,0 +1,19 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.parser;
+
+public class ParsingException extends RuntimeException {
```

Comment:
Is the error location leaved for future development?

Reply:
Yes. This is TODO